tsconfig.json

```diff
@@ -25,7 +27,7 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "commonjs",                                /* Specify what module code is generated. */
```

Comment:
Maybe we should remove these comments from `tsconfig`? It's autogenerated but once you set this up, it will probably stay not touched forever.

Reply:
Personally, I'm ok with autogenerated commented JSON descriptions - Found them helpful many times.